### DISCUSSION TITLE
[Feature]: How to get the value of a constant in a fixed expression

### DISCUSSION THREAD

**Discussion by ccclalala123** [MilesCranmer/PySR#631] [Q&A]
### Feature Request

Hi, thanks for developing the helpful tool.
I want to use pysr to get the values of the constants C1 and C2 in the following fixed expression:
y=(1-C1)*x1/(x2+1)+x3/(C2+1)
x1,x2,x3,y are known data sets.
How should I use pysr to get the constant C1 and C2?
Thanks！

**Comment by MilesCranmer** [comment]
If the functional form is fixed aside from two constants, why not use scipy optimize directly? Or do you mean C1 and C2 are expressions?
